docs/INSTALLATION.md

```diff
@@ -114,6 +115,30 @@ or
 cordova plugin add --save cordova-plugin-facebook4 --variable APP_ID="App ID" --variable APP_NAME="App Name"
 ```
 
+### Co-existing with plugins that use Firebase
+
+Problems may arise when push plugin is used along plugins, that implement Firebase functionality (cordova-plugin-firebase-analytics, for example). Firebase uses `@string/google_app_id`, as does the push plugin, though the value format differs, causing problems like this: `Invalid google_app_id. Firebase Analytics disabled`.
```

Comment:
`along plugins, that implement` -> `along plugins that implement` =  personal suggestion to increase readability.